app/core/utils.coco.coffee

```diff
@@ -3,6 +3,29 @@ slugify = _.str?.slugify ? _.string?.slugify # TODO: why _.string on client and
 isCodeCombat = true
 isOzaria = false
 
+getAnonymizedName = (shouldAnonymous, session) ->
+  if shouldAnonymous and me.get('_id').toString() != session.get('creator')
+    anonymizingUser(session.get('creator'))
+  else
+    session.get('creatorName') or 'Anonymous'
+
+getAnonymizingStatus = (league, supermodel) ->
+  anonymousPlayerName = false
+  if league and anonymousPlayerName = features.enableAnonymization
```

Comment:
oh i doubt. 
```
return unless a
do others
``` 
is the exactly same as 
```
if a
  do others
return
```
right ? only single condition check on `a` ? 😂 oh but yeah, return first may easier for programmers to read .already updated.

Reply:
The point of such refactorings is to keep the "happy path" or ["golden path"](https://github.com/raywenderlich/objective-c-style-guide#golden-path) straightforward and to avoid unnecessary indentation and branches using early `return`s. Here's an old example I've used before of how this would play out in a larger function:

```coffee
      # ... more code above ...
      if codeHasChangedSignificantly and not codeIsAsCast
        if @worker
          workerMessage =
            function: 'transpile'
            spellKey: @spell.spellKey
            source: source

          @worker.addEventListener 'message', (e) =>
            workerData = JSON.parse e.data
            if workerData.function is 'transpile' and workerData.spellKey is @spell.spellKey
              @worker.removeEventListener 'message', arguments.callee, false
              aether.problems = workerData.problems
              aether.raw = source
              finishUpdatingAether(aether)
          @worker.postMessage JSON.stringify(workerMessage)
        else
          aether.transpile source
          finishUpdatingAether(aether)
      else
        finishUpdatingAether(aether)
```

->

```coffee
      # ... more code above ...
      unless codeHasChangedSignificantly and not codeIsAsCast
        return finishUpdatingAether(aether)
      unless @worker
        aether.transpile source
        return finishUpdatingAether(aether)
      workerMessage =
        function: 'transpile'
        spellKey: @spell.spellKey
        source: source
      @worker.addEventListener 'message', (e) =>
        workerData = JSON.parse e.data
        return unless workerData.function is 'transpile' and workerData.spellKey is @spell.spellKey
        @worker.removeEventListener 'message', arguments.callee, false
        aether.problems = workerData.problems
        aether.raw = source
        finishUpdatingAether(aether)
      @worker.postMessage JSON.stringify(workerMessage)
```